package.json

```
@@ -11,19 +11,22 @@
     "dev:server": "cd pyflask && python -m flask run --port 4242",
     "build": "npm run build:app",
     "build:app": "electron-vite build",
+    "build:mac": "npm run build && npm run build:python-unix && npm run build-mac",
+    "build:win": "npm run build && npm run build:python && npm run build-win",
+    "build:python": "python -m PyInstaller --onedir --clean --noconfirm ./pyflask/app.py --distpath ./out/python --add-data ./paths.config.json;. --collect-all pynwb --collect-all hdmf --collect-all ci_info",
+    "build:python-unix": "python -m PyInstaller --onedir --clean --noconfirm ./pyflask/app.py --distpath ./out/python --add-data ./paths.config.json:. --collect-all pynwb --collect-all hdmf --collect-all ci_info",
+    "build-win": "electron-builder build --win --publish never",
+    "build-mac": "electron-builder build --mac --publish never",
+    "build-linux": "electron-builder build --linux --publish never",
     "test": "npm run test:app && npm run test:server",
     "test:app": "vitest run",
     "test:server": "pytest pyflask/tests/ -s",
     "test:coverage": "npm run coverage:app && npm run coverage:server",
     "coverage:app": "vitest run --coverage",
     "coverage:server": "pytest pyflask/tests/ -s --cov=pyflask --cov-report=xml",
-    "build-win": "electron-builder build --win --publish never",
-    "build-mac": "electron-builder build --mac --publish never",
-    "build-linux": "electron-builder build --linux --publish never",
-    "deploy-win": "electron-builder build --win --publish always",
-    "deploy-mac": "electron-builder build --mac --publish always",
-    "deploy-linux": "electron-builder build --linux --publish always",
-    "python-onefile-build": "python -m PyInstaller --onefile --clean ./pyflask/app.py --distpath ./pyflaskdist --add-data ./paths.config.json;. --collect-data jsonschema_specifications --collect-all pynwb --collect-all hdmf --collect-all ci_info",
+    "deploy:win": "electron-builder build --win --publish always",
+    "deploy:mac": "electron-builder build --mac --publish always",
+    "deploy:linux": "electron-builder build --linux --publish always",
```

```suggestion
    "deploy:electron:win": "electron-builder build --win --publish always",
    "deploy:electron:mac": "electron-builder build --mac --publish always",
    "deploy:electron:linux": "electron-builder build --linux --publish always",
```

would then be this for consistency, if accepted